sketch.js

```diff
@@ -194,6 +194,10 @@ function szofajOnClick() {
 	update_incr = picture_delay * 2;
 }
 
+function saveOnClick() {
```

Comment:
`function saveOnClick() {
	let fileFormat = this.elt.innerText;
	showText = false;
	draw();
	saveCanvas("solaris", fileFormat);
	showText = true;
}`

Reply:
Jó ez így. Majd, ha a végleges UI-nál máshogy/nem lesz a text, akkor ki lehet szedni